src/i18n/i18n_module_en.properties

```diff
@@ -2361,3 +2361,6 @@ relationship_label=Custom label for relationship
 note_label=Custom label for note
 tracked_entity_attribute_label=Custom label for tracked entity attribute
 sorting_is_disabled_when_filter_is_applied=Sorting is diabled when a filter is applied
+save_the_program_in_order_to_access_sharing_settings=Save the program in order to access sharing settings
+confirm_delete_icon=Are you sure you want to delete this icon?
+intro_icon=Upload, modify and view icons which can be used assigned to other metadata.
```

Comment:
This was missed when implementing icons. Since this PRs needs to be backported to patch/41, I opted to just sneak it in here 😇